cob_bringup/drivers/base_driver.launch

```diff
@@ -14,20 +13,11 @@
 		<param name="max_trans_velocity" value="1.1"/>	<!-- default: 1.1 m/s -->
 		<param name="max_rot_velocity" value="1.8"/>	<!-- default: 1.8 rad/s -->
 
-		<!-- start underlying components - base_drive_chain -->
-		<node if="$(arg sim)" pkg="cob_base_drive_chain" type="cob_base_drive_chain_sim_node" name="base_drive_chain_node" respawn="false" output="screen">
-			<remap from="/joint_states" to="/base/joint_states"/>
-			<remap from="/base_fr_caster_rotation_controller/command" to="/base/base_fr_caster_rotation_controller/command"/>
-			<remap from="/base_fl_caster_rotation_controller/command" to="/base/base_fl_caster_rotation_controller/command"/>
-			<remap from="/base_br_caster_rotation_controller/command" to="/base/base_br_caster_rotation_controller/command"/>
-			<remap from="/base_bl_caster_rotation_controller/command" to="/base/base_bl_caster_rotation_controller/command"/>
-			<remap from="/base_fr_caster_r_wheel_controller/command" to="/base/base_fr_caster_r_wheel_controller/command"/>
-			<remap from="/base_fl_caster_r_wheel_controller/command" to="/base/base_fl_caster_r_wheel_controller/command"/>
-			<remap from="/base_br_caster_r_wheel_controller/command" to="/base/base_br_caster_r_wheel_controller/command"/>
-			<remap from="/base_bl_caster_r_wheel_controller/command" to="/base/base_bl_caster_r_wheel_controller/command"/>
-		</node>
-		<node unless="$(arg sim)" pkg="cob_base_drive_chain" type="cob_base_drive_chain_node" name="base_drive_chain_node" respawn="false" output="screen">
+		<node pkg="cob_base_drive_chain" type="cob_base_drive_chain_node" name="base_drive_chain_node" respawn="false" output="screen">
 			<remap from="/joint_states" to="/base/joint_states"/>
```

Comment:
The node itself is advertising/publishing to (global) `/joint_states`, right?
I think the remap should be `<remap from="/joint_states" to="joint_states"/>` - using the namespace `base` from the `group`. This would also be better suited for current multi-robot efforts by @ipa-srd-fg